autogen/docs/headings.html.mustache

```diff
@@ -69,7 +69,6 @@
   <span class="block offwhitetext bold nowrap">{{date}}</span>
 </div>
 <div style="margin-bottom: 0em; margin-top: .25em;">
-  <span class="heading-title xsmallfont">Release Notes</span>
```

Comment:
Yes, let's rename "What's new?" to "Release Notes". I agree this was somewhat confusing.